besom-cats/src/main/scala/runtime.scala

```diff
@@ -50,4 +50,30 @@ trait CatsEffectModule extends BesomModule:
   // override def run(program: Context ?=> Output[Exports]): Future[Unit] = ???
 
 object Pulumi extends CatsEffectModule
-export Pulumi.{*, given}
+export Pulumi.{component => _, *, given}
```

Comment:
@lbialy correct me if I'm wrong, but this change looks like a rebase/merge issue?

Reply:
NVMD, fixed, wrong base